airbyte-integrations/connectors/destination-s3/build.gradle

```diff
@@ -8,7 +8,7 @@ plugins {
 airbyteBulkConnector {
     core = 'load'
     toolkits = ['load-s3', 'load-avro', 'load-aws']
-    cdk = '0.1.19'
```

Comment:
Can we pin to a specific version?

Reply:
yep, will pin after I merge https://github.com/airbytehq/airbyte/pull/66521